UI/data/locale/en-US.ini

```diff
@@ -424,6 +424,9 @@ Basic.StatusBar.Delay="Delay (%1 sec)"
 Basic.StatusBar.DelayStartingIn="Delay (starting in %1 sec)"
 Basic.StatusBar.DelayStoppingIn="Delay (stopping in %1 sec)"
 Basic.StatusBar.DelayStartingStoppingIn="Delay (stopping in %1 sec, starting in %2 sec)"
+Basic.StatusBar.DynamicBitrate.Up="Network congestion clearing.\nIncreasing bitrate."
+Basic.StatusBar.DynamicBitrate.Stationary="Network congestion still ongoing.\nBitrate stationary."
+Basic.StatusBar.DynamicBitrate.Down="Network congestion.\nDecreasing bitrate."
```

Comment:
Maybe `Experiencing network congestion.\nDecreasing bitrate.`? I think that might read better.

Reply:
ok will do